src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,209 @@
+package duke;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Represents a list of tasks.
+ */
+public class TaskList {
```

Comment:
Good to see that you have different methods for each command but you can also use only one switch - case statements for all methods instead.